test/com/facebook/buck/android/testdata/android_project/java/com/sample/lib/BUCK.fixture

```diff
@@ -113,3 +113,22 @@ robolectric_test(
         "//res/com/sample/top:top",
     ],
 )
+
+robolectric_test(
+    name = "test_robolectric_runtime_dep",
```

Comment:
You need to add a test in `RobolectricTestRuleIntegrationTest` that uses this target.